src/ViewModel/ArticleSection.php

```diff
@@ -12,6 +12,7 @@ final class ArticleSection implements ViewModel
 {
     const STYLE_DEFAULT = 'default';
     const STYLE_HIGHLIGHTED = 'highlighted';
+    const SEPARETED_CIRCLE = 'circle';
```

Comment:
typo and label suggestion: `const RELATED_LINKS_SEPARATOR_CIRCLE = 'circle';`

Reply:
The variable name has been changed.